addons/bindings/amazondashbutton/readme.md

```diff
@@ -82,3 +82,15 @@ Background discovery is supported. Just press the button in order to put it into
 ## Channels
 
 * Press: Channel for recognizing presses on the Amazon Dash Button
+
+## Example Use
+
+The binding uses a trigger channel which doesn't have to be linked to an item. You can easily use it in your rules like this:
+```
```

Comment:
Could you please add an empty line here as Jekyll will otherwise break the layout on docs.openhab.org?